sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/StandardMetricTests.cs

```diff
@@ -169,6 +169,53 @@ public void ValidateRequestDurationMetricConsumerKind()
             Assert.Equal(StandardMetricConstants.RequestDurationMetricIdValue, metricId);
         }
 
+        [Theory]
+        [InlineData("", false)]
+        [InlineData(null, false)]
+        [InlineData("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36 Edg/124.0.0.0", false)]
+        [InlineData("AlwaysOn/1.0", true)]
+        [InlineData("ALWAYSON/1.0", true)]
+        [InlineData("alwayson/1.0", true)]
+        public void ValidateSyntheticRequest(string? userAgent, bool expectedIsSynthetic)
+        {
+            var activitySource = new ActivitySource(nameof(StandardMetricTests.ValidateSyntheticRequest));
+            var traceTelemetryItems = new List<TelemetryItem>();
+            var metricTelemetryItems = new List<TelemetryItem>();
+
+            var standardMetricCustomProcessor = new StandardMetricsExtractionProcessor(new AzureMonitorMetricExporter(new MockTransmitter(metricTelemetryItems)));
+
+            var traceServiceName = new KeyValuePair<string, object>("service.name", "trace.service");
+            var resourceAttributes = new KeyValuePair<string, object>[] { traceServiceName };
+
+            using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+                .SetSampler(new AlwaysOnSampler())
+                .SetResourceBuilder(ResourceBuilder.CreateDefault().AddAttributes(resourceAttributes))
+                .AddSource(nameof(StandardMetricTests.ValidateSyntheticRequest))
+                .AddProcessor(standardMetricCustomProcessor)
+                .AddProcessor(new BatchActivityExportProcessor(new AzureMonitorTraceExporter(new AzureMonitorExporterOptions(), new MockTransmitter(traceTelemetryItems))))
+                .Build();
+
+            using (var activity = activitySource.StartActivity("Test", ActivityKind.Consumer))
```

Comment:
curious why Consumer kind is used? Standard metrics are calculated for Server/Client as well, and the name of the metric also differs, right?

Reply:
The spec says IsSynthetic is only for requests.
I don't think there was any specific reason `ActivityKind.Consumer` was used here instead of `ActivityKind.Server`. 
We could add both to the test cases, but adding an additional dimension would double the existing cases.

Reply:
Is there a test confirming that we only add `IsSynthetic` dimension for Request Telemetry only? The standard metrics likely have very rigid backend restrictions, potentially dropping entire metric, if additional dimensions are sent!

Also, as per the doc here : https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-metrics/microsoft-insights-components-metrics the synthetic flag is applicable for Dependency, TraceTelemetry as well.

Reply:
Okay, we need to block this PR until I can get some clarifications.